api/src/backend/api/v1/views.py

```diff
@@ -2460,6 +2462,115 @@ def metadata_latest(self, request):
         serializer.is_valid(raise_exception=True)
         return Response(serializer.data)
 
+    @action(
+        detail=True,
+        methods=["get"],
+        url_name="timeline",
+        url_path=r"timeline(?:/(?P<lookback_days>[0-9]+))?",
+    )
+    def timeline(self, request, pk=None, lookback_days=None):
+        """Get CloudTrail timeline events for a resource.
+
+        Returns a timeline of CloudTrail events showing who modified the resource and when.
+        Only available for AWS resources.
+
+        URL parameters:
+        - lookback_days: Number of days to look back (default: 90, max: 365)
+          Usage: /resources/{id}/timeline or /resources/{id}/timeline/30
+        """
+        resource = self.get_object()
+
+        # Check if this is an AWS resource
+        if resource.provider.provider != "aws":
+            return Response(
+                {
+                    "error": "Timeline is only available for AWS resources",
+                    "detail": f"Resource provider is '{resource.provider.provider}', but timeline requires 'aws'",
+                },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        # Get lookback_days parameter (default: 90, max: 365)
+        try:
+            if lookback_days is None:
+                lookback_days = 90
+            else:
+                lookback_days = int(lookback_days)
+
+            if lookback_days < 1 or lookback_days > 365:
+                return Response(
+                    {"error": "lookback_days must be between 1 and 365"},
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+        except ValueError:
+            return Response(
+                {"error": "lookback_days must be a valid integer"},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        try:
+            # Import CloudTrailTimeline here to avoid loading it globally
+            from api.utils import initialize_prowler_provider
+            from prowler.providers.aws.lib.cloudtrail_timeline.cloudtrail_timeline import (
+                CloudTrailTimeline,
+            )
```

Comment:
Why are we importing this here? I mean, this code runs on every call to this endpoint. This should be at the top of the file